backend/src/main/java/net/vidux/camhub/camera/Camera.java

```diff
@@ -0,0 +1,30 @@
+package net.vidux.camhub.camera;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.NaturalId;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.Date;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Builder
+public class Camera {
+  @Id
+  @GeneratedValue
+  private long id;
+  private String name;
+  private String ip;
+  private String firmware;
+  private Date lastSeen;
+  private String password;
+  @NaturalId
+  private String serialNumber;
+}
```

Comment:
- konstruktorok láthatóak mindenhonnan, nincs módunk befolyásolni, hogy jön létre az objektum
 - Date osztály használatát kerüljük, https://programminghints.com/2017/05/still-using-java-util-date-dont/ (ennek utána lehet nézni kicsit jobban is)
 - Null állapotokat ne engedjünk

Reply:
Konstruktorokat elrejtettem, nem csináltam külön factoryt egyelőre. Majd lehet tovább finomhangolni amikor a rawData feldolgozást csináljuk.
Többit megcsináltam